.changeset/flat-cities-say.md

```
@@ -0,0 +1,5 @@
+---
+"task-master-ai": minor
+---
+
+Added api keys page on docs website: docs.task-master.dev/getting-started/api-keys
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)